internal/report.go

```diff
@@ -408,6 +411,12 @@ func GenerateSummary(conv *Conv, r []tableReport, badWrites map[string]int64) st
 	for _, n := range badWrites {
 		badRows += n
 	}
+	numColumns := int32(int(cols))
+	numWarnings := int32(int(warnings))
```

Comment:
I don't remember exactly but what all different warnings do we have? Should we capture those separately?